SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialReceiver.h

```diff
@@ -139,12 +139,10 @@ class USpatialReceiver : public UObject
 
 	void ProcessOrQueueIncomingRPC(const FUnrealObjectRef& InTargetObjectRef, SpatialGDK::RPCPayload InPayload);
 
-	void ResolvePendingOperations_Internal(UObject* Object, const FUnrealObjectRef& ObjectRef);
```

Comment:
Is this now completely internal? (USpatialReceiver::ResolvePendingOperations) I don't know if we prefer to have it declared in the header with private.